benchmark/single-source/ObjectiveCBridging.swift

```diff
@@ -744,3 +747,25 @@ public func run_ASCIIStringFromFile(_ N: Int) {
   #endif
 }
 
+var unicodeStringFromCodable:String? = nil
+var unicodeStringFromCodableDict = [String:Void]()
+public func setup_UnicodeStringFromCodable() {
+  let jsonString = "[\(String(reflecting: string))]"
```

Comment:
`string` isn't defined anywhere. It looks like it was `"Nice string which works rather slöw."` before the force-push. I'm going to revert because I'm not sure what you meant `string` to be.